tap_chargebee/streams/base.py

```diff
@@ -175,23 +175,24 @@ def sync_data(self):
             
             to_write = self.get_stream_data(records)
             
-            if self.ENTITY == 'event':
+            if self.config.get('include_deleted') not in ['false','False', False] and self.ENTITY == 'event':
```

Comment:
Keep self.entity == event in  the nested check

Reply:
DOne